server/src/middleware/validators/userRoutesValidator.ts

```diff
@@ -18,6 +18,7 @@ export const userSignUp = celebrate({
     name: Joi.string().required(),
     email: Joi.string().required(),
     password: Joi.string().required(),
```

Comment:
We have constraints we should be validation here also, such as a couple password things

Reply:
Like what?

Reply:
See the userModel we have a few requirements

Reply:
@witehound this is still pending

Reply:
Those are all the required fields! And now the function works as it should really!

Reply:
@andrew-bierman just checked and I think this issue has been properly resolved

Reply:
Are there any changes in this PR that are not already in andrew_testing ? After syncing branch it does not show any touched files

Reply:
Oh Yh when we had the zod validation issue the last branch that was working was this branch so I reverted to this branch and started building till you finally updated andrew_testing? So most of the lastest brabches I have already have this same changed !